pages/linux/latexmk.md

```diff
@@ -0,0 +1,19 @@
+# latexmk
+
+> Automated LaTeX document generation.
+
+- Compile .tex file to pdf:
+
+`latexmk -pdf file`
```

Comment:
My, I keep forgetting the tokens! Guess I'm too busy looking at the grammar & readability.... I'll have to try and remember them next time :P